_posts/2022-09-8-babelfish-release-2-2-0.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+author: Bill Ramos
+authors: 
+    - Bill Ramos
+comments: true
+title: "Babelfish 2.2.0 is available"
+categories:
+- releases
+---
+
+We’re happy to announce the release of Babelfish 2.2.0, which includes community updates made to PostgreSQL 14.5. This release includes support for a number of features requested by our users:
+
+- Support for old-style function calls with `::` preceding the function name.
```

Comment:
Add the other new features here:

- Support for data migration using the BCP client and the bcp utility
- Support for cross-database stored procedure execution
- Support for `CROSS APPLY` and `OUTER APPLY` (lateral join)
- Support for additional catalog objects

Reply:
Added...